### ISSUE TITLE
SD 2.1 512 model generates brown preview pictures and doesn't train Dreambooth subject

### ISSUE THREAD

**Issue by kakis2** [d8ahazard/sd_dreambooth_extension#911]
### Kindly read the entire form below and fill it out with the requested information.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your
issue will be automatically closed._**

`
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 15e89ef0f6f22f823c19592a401b9e4ee477258c
Dreambooth revision: 9f4d931a319056c537d24669cb950d146d1537b0
SD-WebUI revision: 15e89ef0f6f22f823c19592a401b9e4ee477258c

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.14.dev0 installed.
[+] torch version 1.12.1+cu113 installed.
[+] torchvision version 0.13.1+cu113 installed.
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes
**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes
**Have you updated Dreambooth to the latest revision?**
Yes
**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.**
OK
Model loads SD 2.1 512 model, but while training it only generates brown preview pictures for subject as well as sanity prompt preview. 
My settings work great with SD 1.5 model.
When compiling checkpoints it only goes to 50% 2/4 steps

I have downloaded (v2-inference.yaml.) and renamed yaml file to match model name - 
 Launching Web UI with arguments: --xformers --no-half
Class images are generated with 2.1 512 model
**Provide logs**

Initializing dreambooth training...
Pre-processing aus2: 100%|████████████████████████████████| 12/12 [00:00<00:00, 1200.00it/s]
Pre-processing d57621: 100%|██████████████████████████████| 576/576 [00:03<00:00, 153.66it/s]
Concept requires 48 class images per instance image.
Sorting instance images: 100%|███████████████████████████████████████████| 1/1 [00:00<?, ?it/s]
Nothing to generate.
Replace CrossAttention.forward to use default
Found 576 reg images.
Preparing dataset...
Init dataset!
Preparing Dataset (Without Caching)
Bucket 0 (512, 512, 0) - Instance Images: 12 | Class Images: 576 | Max Examples/batch:   24
Total Buckets 1 - Instance Images: 12 | Class Images: 576 | Max Examples/batch:  24
Processing images...: 100%|███████████████████████████████| 588/588 [00:00<00:00, 10275.79it/s]
Total images / batch: 24, total examples: 24
Processing images...: 100%|███████████████████████████████| 588/588 [00:00<00:00, 10275.79it/s]
Total dataset length (steps): 24
Initializing bucket counter!
  ***** Running training *****
  Num batches each epoch = 24
  Num Epochs = 60
  Batch Size Per Device = 1
  Gradient Accumulation steps = 1
  Total train batch size (w. parallel, distributed & accumulation) = 1
  Text Encoder Epochs: 60
  Total optimization steps = 720
  Total training steps = 1440
  Resuming from checkpoint: False
  First resume epoch: 0
  First resume step: 0
  Lora: None, Adam: False, Prec: fp16
  Gradient Checkpointing: False
  EMA: True
  UNET: True
  Freeze CLIP Normalization Layers: False
  LR: 1e-06
  V2: True
Compiling Checkpoint:  50%|▌| 2/4 [00:04<00:04,  2.43s/it, loss=0.156, loss_avg=0.166, lr=1e-6,Resolution is less than 768 and is a v2 model. Using epsilon mode.
Compiling Checkpoint:  50%|▌| 2/4 [00:05<00:05,  2.78s/it, loss=0.0621, loss_avg=0.159, lr=1e-6Resolution is less than 768 and is a v2 model. Using epsilon mode.
Compiling Checkpoint:  50%|▌| 2/4 [00:05<00:05,  2.63s/it, loss=0.138, loss_avg=0.146, lr=1e-6,Resolution is less than 768 and is a v2 model. Using epsilon mode.
Compiling Checkpoint:  50%|▌| 2/4 [00:04<00:04,  2.40s/it, loss=0.0454, loss_avg=0.0506, lr=1e-Resolution is less than 768 and is a v2 model. Using epsilon mode.
Compiling Checkpoint:  50%|▌| 2/4 [00:05<00:05,  2.56s/it, loss=0.0675, loss_avg=0.098, lr=1e-6Resolution is less than 768 and is a v2 model. Using epsilon mode.
Steps: 100%|█| 1440/1440 [02:02<00:00,  2.06it/s, loss=0.233, loss_avg=0.119, lr=1e-6, vram_usa
Save completed/canceled.
Completed, enabling saves.
Compiling Checkpoint:  50%|▌| 2/4 [00:04<00:04,  2.45s/it, loss=0.233, loss_avg=0.119, lr=1e-6,Resolution is less than 768 and is a v2 model. Using epsilon mode.
Steps:   0%|    | 0/1440 [00:00<?, ?it/s, loss=0.233, loss_avg=0.119, lr=1e-6, vram_usage=20.7]  Training complete (step check).
Cleanup completed.
Steps: 100%|█| 1440/1440 [00:00<00:00, 10125.85it/s, loss=0.233, loss_avg=0.119, lr=1e-6, vram_
Restored system models.
**Environment**

What OS?
Windows 11
If Windows - WSL or native?
Native
What GPU are you using?
4090 24GB

**Screenshots/Config**
![image](https://user-images.githubusercontent.com/99632032/217149781-136645e5-1084-41ed-aeac-c759619b01c3.png)

**Comment by alexdingfelder** [comment]
Getting the same issue

**Comment by focusede** [comment]
I'm getting this as well... any info on it?

**Comment by toomanydev** [comment]
It did the brown previews with me as well. The resulting checkpoints were good and worked normally in the txt2img tab.
I did not have the issue where compiling checkpoints it only gets to 50% 2/4, though, and unfortunately don't know how to help with that.
If anyone gets the brown previews, but gets a checkpoint file out, then test it in txt2img to see if it works after all.

**Comment by chouti** [comment]
I'm getting the same issue, and fine tuning the SD 1.5 would work well.

**Comment by d8ahazard** [comment]
This should be resolved with https://github.com/d8ahazard/sd_dreambooth_extension/releases/tag/1.0.11

Feel free to open a new issue if you're still having problems.
